### DISCUSSION TITLE
Using free tier for my sons's primary school

### DISCUSSION THREAD

**Discussion by olibos** [vercel/community#3227] [Feature Question] [answered]
Hello,

I'm currently working on my free time on the future website for my sons' primary school.

The project will be open source and will use NextJS.

For a better idea of its nature, you can see the current site: https://ecoledesaintjob.be/

I was wondering if the "free plan / hobby / non-commercial" could be used for this?

Thank you,
Olivier

**Comment by amyegan** [comment, marked as answer]
As long as the project stays with the Usage limits and there is no [Commercial Usage](https://vercel.com/docs/concepts/limits/fair-use-policy#commercial-usage) (e.g. the site doesn't take payment and you're not being paid for the site), then the site is fine on a Hobby account. 🙂

**Reply by olibos** [reply]
Great thank you!

I think limits should not be an issue 🙂
